### ISSUE TITLE
Nushell commands return nothing

### ISSUE THREAD

**Issue by nicolaslesquoy** [nushell/nushell#7901]
### Describe the bug

When using some commands, like `ps` or `ls` (same goes for `version`), the command returns nothing.
For instance,
```
/home/nlesquoy〉ls -a | where type == dir | table
╭────┬─────────────────┬──────┬──────────┬───────────────╮
│  # │      name       │ type │   size   │   modified    │
├────┼─────────────────┼──────┼──────────┼───────────────┤
│  1 │ .cache          │ dir  │  4.0 KiB │ 2 weeks ago   │
│  2 │ .cargo          │ dir  │  4.0 KiB │ 2 months ago  │
│  3 │ .config         │ dir  │  4.0 KiB │ 2 days ago    │
...
/home/nlesquoy〉ls
/home/nlesquoy〉
```
But when piping the command like in the description of this issue, the command returns something. This, on the other side, does not work for `ps` for instance.

I installed nushell with `cargo install nu` and I launch it by simply typing `nu` in my terminal. I run **Ubuntu 22.04.1**. 

### How to reproduce

1. open `nushell` with `nu`
2. type `ls` 

### Expected behavior

I expected `nu` to return something for `ls`, especially when using in my `\home`.

### Screenshots

![image](https://user-images.githubusercontent.com/74468941/215350734-ecc95cc1-de5e-4a4b-9c6e-6a2a30167658.png)

### Configuration

`version | transpose key value | to md --pretty` does not return anything.

**Comment by sholderbach** [comment]
Have you set up a custom `config.nu`? My first suspects would be a silent failure in the `display_output` hook or the conditional formatting of table output. I was not able to reproduce this with my existing or the current default config.

**Comment by nicolaslesquoy** [comment]
> Have you set up a custom `config.nu`? My first suspects would be a silent failure in the `display_output` hook or the conditional formatting of table output. I was not able to reproduce this with my existing or the current default config.

I did not configure a custom `config.nu`. The install is fresh.

**Comment by fdncred** [comment]
I was thinking the same thing. Try to comment out this in the config.nu and see what happens.
```
    display_output: {
      if (term size).columns >= 100 { table -e } else { table }
    }
```

**Comment by nicolaslesquoy** [comment]
@fdncred I have some troubles finding the `config.nu` file. I tried `echo $nu.config-path` in `nushell` and `bash` and this didn't produced any results.

**Comment by fdncred** [comment]
in nushell type `config nu` and it should launch the config file in an editor. or you can type `vi $nu.config-path` or `<editor_of_your_choice> $nu.config-path`

**Comment by nicolaslesquoy** [comment]
```
    display_output: {
      $nothing
    }
``` the field `display_output` has this value by default in my config.nu file. I suppose that I should have `if (term size).columns >= 100 { table -e } else { table }` instead of `$nothing` ?

**Comment by nicolaslesquoy** [comment]
After commenting the `display_output` field, the shell works as expected.

**Comment by fdncred** [comment]
`$nothing` will result in everything returning `$nothing`. It should've originally had that if statement that I mentioned above. If you don't want to have that functionality, you have to comment it out vs putting `$nothing` in there.

**Comment by rgwood** [comment]
Hmm. It’s strange (and concerning) that your config.nu ended up like this and we’re not sure why.

**Comment by sholderbach** [comment]
Before https://github.com/nushell/nushell/pull/7172/files (landed with `0.72`) the `display_output` hook was set to `$nothing`. Apparently that was fine then.
